src/util/Sawyer/Assert.h

```diff
@@ -104,7 +105,7 @@ typedef void (*AssertFailureHandler)(const char *mesg, const char *expr, const s
 /** %Optional user callback to handle assertion failures.  If this variable has a non-null value, then that function is called
  *  after the failed assertion message is emitted. This allows the user to terminate the program some other way than the
  *  default @c abort call.  In any case, this function should not return; if it does, then @c abort is called. */
-extern AssertFailureHandler assertFailureHandler;
+ROSE_UTIL_API extern AssertFailureHandler assertFailureHandler;
```

Comment:
The src/util/Sawyer files are copied from a 3rd party library, so they should have nothing that depends on ROSE, such as ROSE_UTIL_API.  So this should probably be SAWYER_EXPORT (defined in src/util/Sawyer/Sawyer.h) instead of ROSE_UTIL_API.